docs/building-apps/wallet/stellar.mdx

```diff
@@ -250,13 +298,29 @@ suspend fun addDeviceKeyPair() {
 }
 ```
 
+```typescript
+const deviceKeyPair = account.createKeypair();
+
+const txBuilder = await stellar.transaction({
+  sourceAddress: newKeyPair,
+  baseFee: 100,
+});
+const modifyAccountTransaction = txBuilder
+  .addAccountSigner(deviceKeyPair, 1)
+  .lockAccountMasterKey()
+  .build();
+newKeyPair.sign(modifyAccountTransaction);
+
+await wallet.stellar().submitTransaction(modifyAccountTransaction);
+```
+
 </CodeExample>
 
 ### Sponsoring Transactions
 
 #### Sponsor Operations
 
-Some operations, that modify account reserves can be [sponsored](https://developers.stellar.org/docs/encyclopedia/sponsored-reserves#sponsored-reserves-operations). For sponsored operations, the sponsoring account will be paying for the reserves instead of the account that being sponsored. This allows you to do some operations, even if account doesn't have enough funds to perform such operations. To sponsor a transaction, simply start a `sponsoring` block:
+Some operations, that modify account reserves can be [sponsored](https://developers.stellar.org/docs/encyclopedia/sponsored-reserves#sponsored-reserves-operations). For sponsored operations, the sponsoring account will be paying for the reserves instead of the account that being sponsored. This allows you to do some operations, even if account doesn't have enough funds to perform such operations. To sponsor a transaction, <LanguageSpecific kt={<span>simply start a <b>sponsoring</b> block:</span>} ts={<span>simply create a building function (describing which operations are to be sponsored) and pass it to the <b>sponsoring</b> method:</span>}/>
```

Comment:
replace `<b>` with `<code>` ?

Reply:
ah didn't know you could do that nice
https://github.com/stellar/stellar-docs/pull/224/commits/19799eb7a0c56ae9f8e49241060db9854ddd667c